lemur/sources/service.py

```diff
@@ -264,20 +264,22 @@ def create(label, plugin_name, options, description=None):
     return database.create(source)
 
 
-def update(source_id, label, options, description):
+def update(source_id, label, plugin_name, options, description):
```

Comment:
Function: Changing the plugin selected required updating the `plugin_name` field as well as the options.